notifications_utils/formatters.py

```diff
@@ -417,7 +417,7 @@ def list(self, body, ordered=True):
                 '<table role="presentation" style="padding: 0 0 20px 0;">'
                 "<tr>"
                 '<td style="font-family: Helvetica, Arial, sans-serif;">'
-                '<ol style="Margin: 0 0 0 20px; padding: 0; list-style-type: decimal;">'
```

Comment:
I wonder if the `margin: 0` are important to keep? in case some mail clients apply different default margins?

Reply:
Hmm you may be right, css in email is pretty finicky. I can put `margin: 0` back for `<ol>` and `<ul>`.